src/core_modules/capture-core/components/WorkingLists/TeiWorkingLists/Setup/hooks/useFiltersOnly.js

```diff
@@ -64,6 +64,19 @@ export const useFiltersOnly = ({
                     },
                 ]
                 : []),
+            {
+                id: MAIN_FILTERS.FOLLOW_UP,
+                type: dataElementTypes.BOOLEAN,
```

Comment:
Hey @eirikhaugstulen ,
Based on the ticket description, the filter should follow the behavior of a "Yes Only" value type. You can achieve this by using `dataElementTypes.TRUE_ONLY` instead.
`options` can be removed and the logic in`transformRecordsFilter` and the `converter` should be adapted for `dataElementTypes.TRUE_ONLY`.
Thanks!

Reply:
Hey! During the discussion we had on the design, we decided to use a boolean instead. This is because we also need the functionality of filtering by enrollments who are not marked for follow up. I have updated the Jira-issue. Thank you!

Reply:
Looks like the API returns a `400` error when selecting both `Yes` and `No`. Is this a bug in the API or should the UI not allow multiple values to be selected for the `followUp` parameter? WDYT?

PS: please also update the detailed design docs with the correct type of filter. Thanks!

https://github.com/dhis2/capture-app/assets/6817073/131a7366-66aa-4fb0-ac91-d19c980735c2

Reply:
Sorry, I was testing with ´multiValueFilter´ to see if the functionality worked and then I forgot to remove it. It should only be possible to select Yes _or_ No!